[Errno 111] Connection refused'))
[2025-09-26 07:00:34,939: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392c8c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,939: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Pass.pkpass_d1478440b8f0a601dc61fc997972045c: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392c8c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,943: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392e210>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,943: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Policy-P000132702713.pdf_0c762fa23bab15c918a170b16ba1f565: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392e210>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,947: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc9139052e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,947: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Invoice_IXITRS744380545357051.pdf_9576c282cb6c3c831d60e60f284c3692: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc9139052e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,951: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392f380>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,951: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Ticket_IXITRS744380545357051.pdf_9bd8ca95c5c43b177ee6933182fa15c2: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392f380>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,954: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392efc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,955: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Pass.pkpass_4a2a26f0021f2a754224a09ed214aa55: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392efc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,958: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392dd60>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,959: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Policy-P000132702222.pdf_a0b3ea32de00fed75d95e51e07f6dd63: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392dd60>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,961: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc913954530>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,962: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Invoice_IXITRS744378683913618.pdf_1246465dfedda1370e8145399e978290: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc913954530>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,965: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc913954350>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,966: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Ticket_IXITRS744378683913618.pdf_cb75a7042dca6a3d4d1b7446f22a2a3d: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc913954350>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,968: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc913955250>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,968: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Pass.pkpass_343e5ecd4b531e9048c23d8f6718e06f: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc913955250>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,972: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Upload error: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392f3e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,972: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [error    ] Failed to process document gmail_Invoice.pdf_fa236cc43d9152e1c4b539a4b5d1945a: HTTPConnectionPool(host='localhost', port=3000): Max retries exceeded with url: /api/v1/documents/upload (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc91392f3e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2025-09-26 07:00:34,974: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:34 [info     ] Historical batch 1 processed   batch_size=133 processed=0
[2025-09-26 07:00:35,175: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:35 [info     ] Processing Gmail batch 2, processed: 133/5000
[2025-09-26 07:00:35,728: WARNING/ForkPoolWorker-3] 2025-09-26 07:00:35 [info     ] Found 100 emails with attachments
